powerline/segments/common/env.py

```diff
@@ -166,6 +169,11 @@ def user(pl, hide_user=None):
 		return None
 	if username == hide_user:
 		return None
+	if hide_domain == True:
+		searchfor = None
+		searchfor = re.search('@', username)
```

Comment:
When searching for documentation on using the .find() and .index() str methods, I see that these are deprecated.  Is that a concern?